### ISSUE TITLE
WoD Dark Portal - Quest cannot continue to allow access

### ISSUE THREAD

**Issue by Obivion** [wormogo/Fixes_for_SPP_Project#54]
On a level 100 player (tested as GM player, and a none GM player), the dark portal quest for WoD is unable to proceed.

Tested sequence:
-Teleported to DarkPortal (Dark portal is red as expected)
-Got quest "The Dark Portal" from Archmage Khadgar (both horde and alliance versions are available)
-Pressed "For Azeroth!" gossip button - Nothing happens, and the buttons remains.

Attempting to walk through the portal once it has gone red, with or without the quest flagged as above results in just walking out the back side and falling down the back. 

GM commanding the quest complete also doesnt help, as this flags to turn in at the map "Assault on the dark portal" which is not in the teleport list and guessing is an instance of some description. (as teleporting to the WoD zone and getting to where it should be isnt possible)

**Comment by wormogo** [comment]
Please check this questline as this worked perfectly. Thank you!

**Comment by bySenom** [comment]
> Please check this questline as this worked perfectly. Thank you!

same

**Comment by Obivion** [comment]
Just rechecked on my server with a fresh 90 player (not a GM account).

Started in stormwind, attempted to get the quest from the "Hero's Call Board" and while the quest is visible at the bottom of the list, accepting it isnt possible. the quest never flags as an active quest.

Visiting Archmage Khadgar by the dark portal results in the same, quest is available, but doesnt allow you to accept, unless you keep backing out, then selecting one of the 2 again and then 1 accepts and adds to the quest log. The gossip option "For Azeroth!" doesn't do anything, regardless of clicking.

Previous test didnt start from the "Hero's Call Board", unless there is another step, I think this is the full quest line for quick access after legion was released.


To test the previous alliance chain, all those quests appear to work from "Attack of the Iron Horde" to going to stormwind for the final hand in. Archmage Khadgar still doesnt have the working gossip button and the portal never becomes active as a tp into WoD zones

I will test out the horde line on a new player in a bit

**Comment by Obivion** [comment]
Found the issue after running a second install and comparing differences.

On my server where the quest wasnt working the following were set in the world.conf

Quests.IgnoreAutoComplete = 1
Quests.IgnoreAutoAccept = 1

This was to correct something else when I first set it up, but I cant think what it was now.

Either way, resetting both back to the defaults of 0 allows the completion of the quest line and for the gossip button "For Azeroth!" to function as expected.

I will close this issue, and if I ever remember or run into the reason I changed those setting log an issue for that.
